game-using-jsDOM/script.js

```diff
@@ -0,0 +1,219 @@
+// Array of words that we will use for the game 
+const words = ['penguins', 'elephant', 'koala', 'synonym', 'peanut',
+    'english', 'ruby', 'react', 'friday', 'airplane', 'banana', 'birthday',
+    'waterfall', 'trampoline', 'window', 'president', 'basketball', 'hiccup',
+    'cheerleader', 'kangaroo', 'groundhog', 'soccer', 'handball', 'caterpillar',
+    'shoelaces', 'hungry', 'fishing', 'monster'];
+
+/**
+ * Gets a random word from the words array declared on line 2
+ * @returns a word that has been randomly selected from the words_array
+ */
+function getWord() {
+    return (words[Math.floor(Math.random() * words.length)]);
+}
+
+// grab and store the body element tag
+let body = document.getElementsByTagName("body")[0];
+
+/**
+ * Creates new elements of the type passed by the parameter elementType
+ * Updates elements innerHTML to be text arg and sets class name 
+ * @param {string} elementType 
+ * @param {string} text 
+ * @param {string} className 
+ * @returns the created element 
+ */
+function generateElements(elementType, text, className) {
+    let element = document.createElement(elementType);
+    element.innerHTML = text;
+    element.className = className;
+    return element;
+}
+
+/**
+ * Creates a new button element and modifies the button text 
+ * to the buttonName arg and sets a class attribute 
+ * @param {string} buttonName 
+ * @param {string} className 
+ * @returns the created button 
+ */
+function generateButtons(buttonName, className) {
+    let button = document.createElement("button");
+    button.type = "button";
+    button.className = className;
+    button.innerText = buttonName;
+    return button;
+}
+
+/**
+ * Toggles a class to the startButton that hides the button 
+ * Calls the generateGame() function to create the game
+ */
+function startClicked() {
```

Comment:
Do you know what happen if you call this function more than once time?

May would be better if instead of `.toggle()` we use `.remove()` and check that out if the game is already started

Reply:
I decided not to use toggle since startClicked() is only called once when the "startButton" at the beginning is clicked, after it is hidden so that you can no longer access the Start Button or startClicked function. I don't know if that makes sense hahha

Reply:
Did you try double clicking the button just to double check?

Reply:
It is not possible to double click because as soon as you click once it instantly disappears. I have tried doing it really quick but once I click it one time it disappears. I have also discussed this with Margaret and she also agrees!